bin/check-dns.rb

```diff
@@ -184,7 +185,7 @@ def check_results(entries)
         # consistency reasons
         if config[:type] == 'A' || config[:type] == 'AAAA'
           check_ips(entries)
-        # non ip type
+          # non ip type
```

Comment:
this should not be indented it is explaining that the else is for non ip type records.